communications/blogs/turing_joins_binder_federation.md

```diff
@@ -0,0 +1,102 @@
+# The Turing Institute invests in Open Infrastructure for Reproducible Research
```

Comment:
Maybe capitalise "Invests" here? I get why the capitalisation is the way it is, but the small i looks a little odd on its own!

Reply:
```suggestion
# The Turing Institute Invests in Open Infrastructure for Reproducible Research
```